src/app/settings/admin/presets/dialog/steps/settings/provider/openstack/template.html

```diff
@@ -15,59 +15,98 @@
 -->
 <form [formGroup]="form"
       fxLayout="column"
-      fxLayoutGap="16px">
-  <mat-form-field fxFlex>
-    <mat-label>Username</mat-label>
-    <input matInput
-           [formControlName]="Controls.Username"
-           [name]="Controls.Username"
-           type="text"
-           autocomplete="off"
-           required>
-    <mat-error *ngIf="form.get(Controls.Username).hasError('required')">
-      Username is <strong>required</strong>.
-    </mat-error>
-  </mat-form-field>
+      fxLayoutGap="8px">
+  <mat-hint>Either User or Application credentials must be provided.</mat-hint>
```

Comment:
Can you move it under the form field like we do it in other cases? Also, make sure that the font size is correct.

Reply:
I'm removing this since IMHO it's not required anymore. This is a remanent from my previous approach where we were allowing users to enter both credentials as well.